cmd/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"os"
+
```

Comment:
usually we have `cmd/<app name>/main.go` and so on, maybe we should keep the same structure

Reply:
Right, good catch, moved it to `cmd/proxy/main.go`